sdv/data_processing/data_processor.py

```diff
@@ -346,115 +346,234 @@ def _get_transformer_instance(self, sdtype, column_metadata):
 
         return deepcopy(transformer)
 
-    def _create_config(self, data):
-        sdtypes = {}
-        transformers = {}
+    def _handle_multi_column(self, column, data):
```

Comment:
can we rename all of these to `_get_{descriptor}_config`? So this would be `_get_multi_column_config`